test/xpu_api/utilities/pairs/pair.astuple/get_const_pair_as_tuple.pass.cpp

```diff
@@ -0,0 +1,58 @@
+// -*- C++ -*-
```

Comment:
The name of the test is really strange. As for me, tuple should not appear in this name.

Reply:
Initial name was `get_const.pass.cpp` but we had conflict of test names.
In this case I have simple added something from parent folder name.

Reply:
I was confused because this test is not about tuples. I checked one more time and this test is mostly duplicated with `make_pair.pass.cpp`. I propose to remove this one.